src/script/view_model/content/PreferencesAccountViewModel.ts

```diff
@@ -250,6 +253,28 @@ export class PreferencesAccountViewModel {
     }
   };
 
+  changeEmail = async (data: unknown, event: ChangeEvent<HTMLInputElement>): Promise<void> => {
+    try {
+      const enteredEmail = event.target.value;
+
+      await this.userRepository.changeEmail(enteredEmail);
+      amplify.publish(WebAppEvents.WARNING.MODAL, ModalsViewModel.TYPE.ACKNOWLEDGE, {
+        text: {
+          message: 'We have sent you a link to verify your new email',
+          title: 'Check your inbox',
```

Comment:
Will we get translation for this?

Reply:
For sure.. just placeholders for now.. in the message we should also include the `enteredEmail`.. Something among the lines: 

> We sent a confirmation email to `example@wire.com`. Please check your email and follow the link to verify it.

Reply:
Then let's add a `TODO` :)